admin/controllers/questions.js

```diff
@@ -3,14 +3,18 @@
 const apiResponse = require('./api-response')
 const checkFormService = require('../services/check-form.service')
 const checkStartService = require('../services/check-start.service')
+const checkWindowService = require('../services/check-window.service')
+const config = require('../config')
 const configService = require('../services/config.service')
 const jwtService = require('../services/jwt.service')
+const moment = require('moment')
 const pupilAuthenticationService = require('../services/pupil-authentication.service')
+const pupilDataService = require('../services/data-access/pupil.data.service')
 const pupilLogonEventService = require('../services/pupil-logon-event.service')
-const checkWindowService = require('../services/check-window.service')
 const R = require('ramda')
 const winston = require('winston')
 
+
```

Comment:
Linting error: More than 1 blank line not allowed.

Reply:
This should now be fixed as i have re-enabled codacy and standardJS should enforce this by default.